starknet-foundry/crates/cast/src/starknet_commands/multicall.rs

```diff
@@ -36,12 +38,29 @@ struct InvokeCall {
 #[derive(Args)]
 #[command(about = "Execute multiple calls at once", long_about = None)]
 pub struct Multicall {
-    #[clap(short = 'p', long = "path")]
-    /// Path to a .toml file containing the multi call specification
-    pub path: String,
+    #[clap(subcommand)]
+    pub command: Command,
 }
 
-pub async fn multicall(
+#[derive(Debug, Subcommand)]
+pub enum Command {
+    Run {
+        /// path to the toml file with declared operations
+        #[clap(short = 'p', long = "path")]
+        path: String,
```

Comment:
Can't we use UTF8PathBuf here?

Reply:
good idea thx

Reply:
[use path](https://github.com/foundry-rs/starknet-foundry/pull/98/commits/e1f1b159870caddedb1f12e5625f060088c7d96d)